beacon-chain/rpc/beaconv1/pool.go

```diff
@@ -56,3 +78,24 @@ func (bs *Server) ListPoolVoluntaryExits(ctx context.Context, req *ptypes.Empty)
 func (bs *Server) SubmitVoluntaryExit(ctx context.Context, req *ethpb.SignedVoluntaryExit) (*ptypes.Empty, error) {
 	return nil, errors.New("unimplemented")
 }
+
+func mapAttestation(attestation *eth.IndexedAttestation) *ethpb.IndexedAttestation {
+	a := &ethpb.IndexedAttestation{
+		AttestingIndices: attestation.AttestingIndices,
```

Comment:
Any chance there could be a bad attestation that makes it into this and causes the code to panic?

Reply:
+1 

```go
if attestation == nil {
  return nil
}
```

Reply:
I don't think this is possible. A bad attestation would never result in a slashing, right?